server

```diff
@@ -180,6 +180,21 @@ setup_install() {
   fi
 }
 
+get_full_version () {
```

Comment:
Since this is a new function, you don't need the extra commits , please squash them to one.